src/main/kotlin/_3_racing_car/service/Racing.kt

```diff
@@ -1,7 +1,12 @@
 package _3_racing_car.service
 
+import _3_racing_car.service.dto.RacingInputDto
+import _3_racing_car.service.dto.RacingOutputDto
+
 class Racing {
-    fun run(racingInput: RacingInputDto): List<Int> = racingInput.tracks.map {
-        it.run(racingInput.times)
-    }
+    fun run(racingInput: RacingInputDto) =
+        racingInput.tracks.associate { track ->
+            val racer = track.run(racingInput.times)
+            racer.name to racer.location
+        }.let(::RacingOutputDto)
```

Comment:
```suggestion
    fun run(racingInput: RacingInputDto): RacingOutputDto {
    
        val racerResults: Map<String, Int> = racingInput.tracks.associate { track ->
            val racer = track.run(racingInput.times)
            racer.name to racer.location
        }

        return RacingOutputDto(racerResults)
    }
```

한번에 처리하는것도 좋겠지만, 이렇게 `associate`의 리턴값이 무엇이고, 그것을 `RacingOutputDto`로 변경한다는게 잘 나타나면 가독성이 더 좋아보입니다.